dialyzer.ignore-warnings

```diff
@@ -1,3 +1,25 @@
 apps/omg_performance/
 apps/omg/test
 apps/omg_eth/test
+
+# TODO
+apps/omg_eth/
```

Comment:
I would suggest we're very specific which errors we're ignoring. This would capture every issue, not only those coming because of added `specdiffs`.